v2/cf-nodejs-logging-support/src/lib/config/jwt-service.ts

```diff
@@ -0,0 +1,50 @@
+import Logger from "../logger/logger";
```

Comment:
the jwt service token should maybe reside in its own pr

Reply:
it originally resides in pr https://github.com/SAP/cf-nodejs-logging-support/pull/124 that should be merged soon, I pulled that branch to already resolve some conflicts.

Reply:
> it originally resides in pr #124 that should be merged soon, I pulled that branch to already resolve some conflicts.

The before mentioned pr has been already merged and the rework branch was pulled into this pr. Now it should be more clear which changes have been really made for this branch.